doc/source/ray-air/doc_code/accelerate_trainer.py

```diff
@@ -0,0 +1,73 @@
+import torch
+import torch.nn as nn
+
+from accelerate import Accelerator
+
+import ray
+from ray.air import session, Checkpoint
+from ray.train.huggingface.accelerate import AccelerateTrainer
+from ray.air.config import ScalingConfig
+
+
+# If using GPUs, set this to True.
+use_gpu = False
+
+
+input_size = 1
+layer_size = 15
+output_size = 1
+num_epochs = 3
```

Comment:
turn these into flags?
alternatively, they should be ALL_CAP if these are constants

Reply:
so they are all wrong then: https://peps.python.org/pep-0008/#constants